### ISSUE TITLE
Windows GUI agent doesn't work on large screen resolutions

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#1896]
#### Qubes OS version (e.g., `R3.1`): 
3.0, 3.1

#### Affected TemplateVMs (e.g., `fedora-23`, if applicable):
Windows, with Qubes Windows Tools installed

---

### Expected behavior: 
Windows VM starts, applications are visible

### Actual behavior: 
Windows VM starts, but no application window is visible. Qrexec does work (`qvm-run -p win7 cmd.exe` gives shell).

### Steps to reproduce the behavior:
1. Set dom0 resolution to some high value (for example 2880x1620)
2. Start Windows VM

You can get resolution bigger than physical screen size by using  `xrandr --output OUTPUT_NAME --panning 2880x1620`

---

#### Related issues:
https://groups.google.com/d/msgid/qubes-users/20160329002536.GE14681%40mail-itl
https://github.com/QubesOS/qubes-issues/issues/1704

**Comment by axon-qubes** [comment]
I strongly suspect this to be a duplicate of #1550.

I just installed a fresh Windows 7 HVM template, experienced the issue described in #1550, then confirmed that Qrexec still work (`qvm-run -p win7 cmd.exe` gives me the Windows VM's shell). My resolution is (much) higher than 2880x1620.

**Comment by marmarek** [comment]
[Interesting observation](https://groups.google.com/d/msgid/qubes-users/e1d494c9-821b-42bb-8f87-75f2c4d41a61%40googlegroups.com):
> I have dual monitor setup (1920x1200 Displayport-0 and 1920x1080 DVI-0),
but  I've managed  to install GUI tools (as you did) by only setting dom0
resolution the same as in Windows and detaching the other monitor. As soon
I add the other monitor, Windows wouldn't start no matter if seamless mode
is on or off.   HOWEVER if I add the 2nd monitor only AFTER starting
Windows, it works nicely and I can use Windows both in full screen mode (in
1st or 2nd monitor) or in seamless mode. Also switching between seamless
and normal mode from VM Manager is possible in real-time :)

> Some things I noticed (only 1 monitor in use):
- some of the Windows Updates have incompatibilities with QWT. Whether
Network Tools are installed or not, DHCP would not work, but manual
configuration would resolve situation.
- Seamless mode starts anyway even if it is not enabled in VM Manager when
the VM is booted the first time after GUI tools installation. After that,
it boots as it is configured in VM manager.
- starting in seamless mode is very erratic (about 50% of the times the
startup fails and restart is needed).
- starting without seamless mode works slightly better (starts in full
screen by default but works as windowed as well), but can get stuck as well
during boot
- shutdown from VM Manager does not work (seamless or not), but "shutdown
/s /t 0" from windows command line works even in seamless mode
